bin/plot_krun_results

```diff
@@ -727,6 +873,11 @@ def create_cli_parser():
                         help=('Name of the PDF file to write to. If no file is '
                               'specified, charts will be displayed interactively '
                               'and not written to disk.'))
+    parser.add_argument('--wallclock-only',
```

Comment:
I reckon later, we may find ourselves needing (e.g.) only core 3's cycle graph. Cross that bridge when we fall off of it.